<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="TVMATP310440M" name="Test" tests="12" failures="12" timestamp="11 Nov 2016 10:25:20 GMT" time="2183.192" errors="0">
  <testcase name="INV_NSI_010_Add_Prep_Item_CustomCat_CategoryAsLocation" time="265.492" classname="com.uom.tests.NonSyscoPrepItems">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at com.framework.reporting.FrameworkReporter.fail(FrameworkReporter.java:32)
	at com.framework.frameworkFunctions.LibraryPage.clickElement(LibraryPage.java:104)
	at com.uom.pages.common.GenericPage.tapNextButton(GenericPage.java:454)
	at com.uom.tests.NonSyscoPrepItems.INV_NSI_010_Add_Prep_Item_CustomCat_CategoryAsLocation(NonSyscoPrepItems.java:1106)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
... Removed 13 stack frames]]>
    </failure>
  </testcase> <!-- INV_NSI_010_Add_Prep_Item_CustomCat_CategoryAsLocation -->
  <testcase name="INV_NSI_010_Add_Prep_Item_CustomCat_CategoryAsLocation" time="269.06" classname="com.uom.tests.NonSyscoPrepItems">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at com.framework.reporting.FrameworkReporter.fail(FrameworkReporter.java:32)
	at com.framework.frameworkFunctions.LibraryPage.clickElement(LibraryPage.java:104)
	at com.uom.pages.common.GenericPage.tapNextButton(GenericPage.java:454)
	at com.uom.tests.NonSyscoPrepItems.INV_NSI_010_Add_Prep_Item_CustomCat_CategoryAsLocation(NonSyscoPrepItems.java:1106)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
... Removed 13 stack frames]]>
    </failure>
  </testcase> <!-- INV_NSI_010_Add_Prep_Item_CustomCat_CategoryAsLocation -->
  <testcase name="INV_NSI_010_Add_Prep_Item_CustomCat_CategoryAsLocation" time="264.742" classname="com.uom.tests.NonSyscoPrepItems">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at com.framework.reporting.FrameworkReporter.fail(FrameworkReporter.java:32)
	at com.framework.frameworkFunctions.LibraryPage.clickElement(LibraryPage.java:104)
	at com.uom.pages.common.GenericPage.tapNextButton(GenericPage.java:454)
	at com.uom.tests.NonSyscoPrepItems.INV_NSI_010_Add_Prep_Item_CustomCat_CategoryAsLocation(NonSyscoPrepItems.java:1106)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
... Removed 13 stack frames]]>
    </failure>
  </testcase> <!-- INV_NSI_010_Add_Prep_Item_CustomCat_CategoryAsLocation -->
  <testcase name="INV_NSI_012_Add_NonSysco_Item_DefaultCat_DefaultLoc_SupplierList_LocQtyUOM" time="231.938" classname="com.uom.tests.NonSyscoPrepItems">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at com.framework.reporting.FrameworkReporter.fail(FrameworkReporter.java:32)
	at com.uom.pages.common.ProductPage.verifyProductDetails(ProductPage.java:231)
	at com.uom.tests.NonSyscoPrepItems.INV_NSI_012_Add_NonSysco_Item_DefaultCat_DefaultLoc_SupplierList_LocQtyUOM(NonSyscoPrepItems.java:1369)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
... Removed 13 stack frames]]>
    </failure>
  </testcase> <!-- INV_NSI_012_Add_NonSysco_Item_DefaultCat_DefaultLoc_SupplierList_LocQtyUOM -->
  <testcase name="INV_NSI_012_Add_NonSysco_Item_DefaultCat_DefaultLoc_SupplierList_LocQtyUOM" time="4.398" classname="com.uom.tests.NonSyscoPrepItems">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at com.framework.reporting.FrameworkReporter.fail(FrameworkReporter.java:32)
	at com.framework.frameworkFunctions.LibraryPage.isElementPresentAfterWait(LibraryPage.java:416)
	at com.uom.pages.common.HomePage.logoutAfterCheck(HomePage.java:271)
	at com.uom.components.Components.login(Components.java:85)
	at com.uom.tests.NonSyscoPrepItems.INV_NSI_012_Add_NonSysco_Item_DefaultCat_DefaultLoc_SupplierList_LocQtyUOM(NonSyscoPrepItems.java:1302)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
... Removed 13 stack frames]]>
    </failure>
  </testcase> <!-- INV_NSI_012_Add_NonSysco_Item_DefaultCat_DefaultLoc_SupplierList_LocQtyUOM -->
  <testcase name="INV_NSI_012_Add_NonSysco_Item_DefaultCat_DefaultLoc_SupplierList_LocQtyUOM" time="229.677" classname="com.uom.tests.NonSyscoPrepItems">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at com.framework.reporting.FrameworkReporter.fail(FrameworkReporter.java:32)
	at com.uom.pages.common.ProductPage.verifyProductDetails(ProductPage.java:231)
	at com.uom.tests.NonSyscoPrepItems.INV_NSI_012_Add_NonSysco_Item_DefaultCat_DefaultLoc_SupplierList_LocQtyUOM(NonSyscoPrepItems.java:1369)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
... Removed 13 stack frames]]>
    </failure>
  </testcase> <!-- INV_NSI_012_Add_NonSysco_Item_DefaultCat_DefaultLoc_SupplierList_LocQtyUOM -->
  <testcase name="INV_NSI_015_Error_Message_ChangeUOMAttriubte_ForNonSyscoItem" time="4.617" classname="com.uom.tests.NonSyscoPrepItems">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at com.framework.reporting.FrameworkReporter.fail(FrameworkReporter.java:32)
	at com.framework.frameworkFunctions.LibraryPage.isElementPresentAfterWait(LibraryPage.java:416)
	at com.uom.pages.common.HomePage.logoutAfterCheck(HomePage.java:271)
	at com.uom.components.Components.login(Components.java:85)
	at com.uom.tests.NonSyscoPrepItems.INV_NSI_015_Error_Message_ChangeUOMAttriubte_ForNonSyscoItem(NonSyscoPrepItems.java:1674)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
... Removed 13 stack frames]]>
    </failure>
  </testcase> <!-- INV_NSI_015_Error_Message_ChangeUOMAttriubte_ForNonSyscoItem -->
  <testcase name="INV_NSI_015_Error_Message_ChangeUOMAttriubte_ForNonSyscoItem" time="181.4" classname="com.uom.tests.NonSyscoPrepItems">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at com.framework.reporting.FrameworkReporter.fail(FrameworkReporter.java:32)
	at com.uom.pages.common.LocationsPage.verifyItemPresentInLocation(LocationsPage.java:1478)
	at com.uom.tests.NonSyscoPrepItems.INV_NSI_015_Error_Message_ChangeUOMAttriubte_ForNonSyscoItem(NonSyscoPrepItems.java:1705)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
... Removed 13 stack frames]]>
    </failure>
  </testcase> <!-- INV_NSI_015_Error_Message_ChangeUOMAttriubte_ForNonSyscoItem -->
  <testcase name="INV_NSI_015_Error_Message_ChangeUOMAttriubte_ForNonSyscoItem" time="187.173" classname="com.uom.tests.NonSyscoPrepItems">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at com.framework.reporting.FrameworkReporter.fail(FrameworkReporter.java:32)
	at com.uom.pages.common.LocationsPage.verifyItemPresentInLocation(LocationsPage.java:1478)
	at com.uom.tests.NonSyscoPrepItems.INV_NSI_015_Error_Message_ChangeUOMAttriubte_ForNonSyscoItem(NonSyscoPrepItems.java:1705)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
... Removed 13 stack frames]]>
    </failure>
  </testcase> <!-- INV_NSI_015_Error_Message_ChangeUOMAttriubte_ForNonSyscoItem -->
  <testcase name="INV_NSI_015a_Error_Message_ChangeUOMAttriubte_ForPrepItem" time="4.335" classname="com.uom.tests.NonSyscoPrepItems">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at com.framework.reporting.FrameworkReporter.fail(FrameworkReporter.java:32)
	at com.framework.frameworkFunctions.LibraryPage.isElementPresentAfterWait(LibraryPage.java:416)
	at com.uom.pages.common.HomePage.logoutAfterCheck(HomePage.java:271)
	at com.uom.components.Components.login(Components.java:85)
	at com.uom.tests.NonSyscoPrepItems.INV_NSI_015a_Error_Message_ChangeUOMAttriubte_ForPrepItem(NonSyscoPrepItems.java:1759)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
... Removed 13 stack frames]]>
    </failure>
  </testcase> <!-- INV_NSI_015a_Error_Message_ChangeUOMAttriubte_ForPrepItem -->
  <testcase name="INV_NSI_015a_Error_Message_ChangeUOMAttriubte_ForPrepItem" time="4.317" classname="com.uom.tests.NonSyscoPrepItems">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at com.framework.reporting.FrameworkReporter.fail(FrameworkReporter.java:32)
	at com.framework.frameworkFunctions.LibraryPage.isElementPresentAfterWait(LibraryPage.java:416)
	at com.uom.pages.common.HomePage.logoutAfterCheck(HomePage.java:271)
	at com.uom.components.Components.login(Components.java:85)
	at com.uom.tests.NonSyscoPrepItems.INV_NSI_015a_Error_Message_ChangeUOMAttriubte_ForPrepItem(NonSyscoPrepItems.java:1759)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
... Removed 13 stack frames]]>
    </failure>
  </testcase> <!-- INV_NSI_015a_Error_Message_ChangeUOMAttriubte_ForPrepItem -->
  <testcase name="INV_NSI_015a_Error_Message_ChangeUOMAttriubte_ForPrepItem" time="105.089" classname="com.uom.tests.NonSyscoPrepItems">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at com.framework.reporting.FrameworkReporter.fail(FrameworkReporter.java:32)
	at com.uom.pages.common.LocationsPage.verifyItemPresentInLocation(LocationsPage.java:1478)
	at com.uom.tests.NonSyscoPrepItems.INV_NSI_015a_Error_Message_ChangeUOMAttriubte_ForPrepItem(NonSyscoPrepItems.java:1789)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
... Removed 13 stack frames]]>
    </failure>
  </testcase> <!-- INV_NSI_015a_Error_Message_ChangeUOMAttriubte_ForPrepItem -->
</testsuite> <!-- Test -->
